airbyte-webapp/src/components/common/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -15,12 +15,14 @@ export interface ConfirmationModalProps {
   onSubmit: () => void;
   submitButtonDataId?: string;
   cancelButtonText?: string;
+  extra?: React.ReactNode;
 }
 
 export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
   onClose,
   title,
   text,
+  extra = null,
```

Comment:
Just curious, why do we set `null` as a default value?

Reply:
This may not be necessary

Reply:
I think `undefined` will render out the same as `null`